relay-profiling/src/transaction_metadata.rs

```diff
@@ -35,11 +43,7 @@ pub struct TransactionMetadata {
 
 impl TransactionMetadata {
     pub fn valid(&self) -> bool {
-        !self.id.is_nil()
-            && !self.name.is_empty()
-            && !self.trace_id.is_nil()
-            && self.relative_cpu_start_ms <= self.relative_cpu_end_ms
-            && self.relative_start_ns < self.relative_end_ns
+        !self.id.is_nil() && !self.name.is_empty() && !self.trace_id.is_nil()
```

Comment:
Could you add a test case that shows that a JSON payload without `relative_start_ns` / `end_ns` deserializes correctly and is now `valid`?